### PR TITLE
Create amd.json

### PR DESCRIPTION
**Pull request by btegs**
Initial commit, needing feedback on where to place this and if this is the right method to add

### CODE REVIEW

File: data/share/tweaks/amd.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+	"id": "amd_radeon_aco_vulkan",
+	"name": "AMD: Radeon ACO Vulkan Compiler",
+	"description": "Enable ACO based Vulkan shader compiler for RADV instead of LLVM",
+	"url": "https://github.com/mesa3d/mesa",
```
Comment (tkashkin) [inline data/share/tweaks/amd.json]: Maybe this link should explain what this option does instead of linking to mesa source.

Comment (btegs) [reply data/share/tweaks/amd.json]: @tkashkin - I've mainly been following Phoronix with artices like this: https://www.phoronix.com/scan.php?page=article&item=radv-aco-feb2020

**Comment by tkashkin** [comment]
@btegs I have a few questions:
* Is this option applicable to native Vulkan games or is it only useful for Wine/DXVK/Proton?
* Do you know any other potentially useful AMD-specific options which can be added to GameHub?
* What does `lspci -k | grep -EA3 'VGA|3D|Display'` output on your system?
I want to check loaded kernel modules to automatically disable or hide tweaks that are not applicable to user's GPU.

**Comment by btegs** [comment]
@tkashkin - I believe this works for all Vulkan games. Both native and Wine/DXVK/Proton. As for other AMD options, I don't have any others at this time. Here's my output from lspci for my Radeon 580 using Mesa from the https://launchpad.net/~kisak/+archive/ubuntu/kisak-mesa PPA in Ubuntu 19.10:

```
09:00.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Ellesmere [Radeon RX 470/480/570/570X/580/580X/590] (rev e7)
	Subsystem: Tul Corporation / PowerColor Ellesmere [Radeon RX 470/480/570/570X/580/580X/590]
	Kernel driver in use: amdgpu
	Kernel modules: amdgpu
```


D

**Pull request merged**